lib/db.js

```diff
@@ -512,19 +514,30 @@ module.exports = (
       lastAccessTime: token.lastAccessTime,
       createdAt: token.createdAt,
       isMemoryToken: true
-    }]
+    }
     let sessionTokens = []
+    return getGeoData(ip)
```

Comment:
Running locally with a 100-200 ips in the same country ranges from ~18
Running locally with 10 ips from 10 different countries was also ~18ms.

Would that be okay to move on with? I can add to docs in fxa-geodb. Not sure if there's a better way to test its performance?